src/routes/docs-examples/extend/virtual-masonry/+page.server.ts

```
@@ -0,0 +1,32 @@
+  height: number;
+  alt: string;
+  author: string;
+}
+
+export const load: PageServerLoad = async ({ fetch }) => {
+  try {
+    const res = await fetch('/api/images');
+    if (!res.ok) throw new Error('Failed to fetch images');
+
+    const data = await res.json();
+    const images: Image[] = data.map((img: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)